cairo/ethereum/cancun/fork_types.cairo

```diff
@@ -181,3 +183,10 @@ func Account__eq__(a: OptionalAccount, b: OptionalAccount) -> bool {
 
     return code_eq;
 }
+
+func Address_from_felt{range_check_ptr, bitwise_ptr: BitwiseBuiltin*}(value: felt) -> Address {
+    let value_u256 = felt_to_uint256(value);
+    tempvar value_U256 = U256(new U256Struct(value_u256.low, value_u256.high));
+    let address = U256_to_be_bytes20(value_U256);
+    return address;
+}
```

Comment:
I think the issue is that here the address passed from the public_key_point... fn is in big-endian no?